components/dashboard-deployment/src/main/java/org/wso2/carbon/dashboard/deployment/DashboardDeployer.java

```diff
@@ -168,6 +168,10 @@ private void deploy(List<Artifact> artifacts) throws DashboardDeploymentExceptio
                         if (file.isDirectory()) {
                             String themeName = file.getName();
                             String storePath = getArtifactPath(DashboardConstants.THEME_TYPE);
+                            File themeFolder = new File(storePath);
+                            if (!themeFolder.exists()) {
```

Comment:
Fixed in commit d5d4041b204edcb7e3591d70d45e37b2aa80c8cd
https://github.com/wso2/carbon-dashboards/pull/452/commits/d5d4041b204edcb7e3591d70d45e37b2aa80c8cd#diff-e4c6614aad35a23ad0e32e2ae4770378R168